### ISSUE TITLE
[Quest][Classic] Seeking Spiritual Aid

### ISSUE THREAD

**Issue by Artanis5** [dalaranwow/dalaran-wow#4722]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

This quest (http://www.wowhead.com/quest=5158/deprecated-seeking-spiritual-aid) requires player to go from felwood to barrens and upon completion of the quest there is triggered a cutscene.
Basically you complete quest with Islen Waterseer (http://www.wowhead.com/npc=5901/islen-waterseer)
She runs to the water.
Then a water elemental (Manifestation of Water, http://www.wowhead.com/npc=11256/manifestation-of-water) is spawned.
They have a chat which is now currently:
Manifestation of Water: NO!!! It puts the tanning in the basket, or it gets the mallet again!
Islen Waterseer: Thank you, great spirit. Thank you!

Btw, that line is actually of Gordok Bushwacker (http://www.wowhead.com/npc=14351/gordok-bushwacker) invovlved in creating the Ogre Suit in Dire Maul. (you climb to the basket (http://www.wowhead.com/object=179499/ogre-tannin-basket) and take the tanning (http://www.wowhead.com/item=18240/ogre-tannin), he spawns and says it)

**Expected behaviour**: Tell us what should happen instead

Now according to wowwiki the conversation should be as follows:
Islen Waterseer walks to the shore 
Islen Waterseer says: Spirits of Water, I give you praise, and I beg of you a favor. You have heard <name>'s please, and I trust [her/his] intent is noble. Please, will you aid us? 
A huge water elemental appears. 
Manifestation of Water says: [ Kalimag ] Ma reth bromo zoln kilagrin dra ma zoern tu ko fraht ko tadrom Ma krin drinor zoln drinor Ma krin kan aasrugel korsul. 
Islen Waterseer says: Thank you, great spirit. Thank you!

**Steps to reproduce the problem**:

1. Complete quest and wait
2. ????
3. Profit

**Include proofs for this behaviour**

http://i.imgur.com/nUeWIjC.jpg?1

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

Thanks for looking into it, have a nice day.

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
> NO!!! It puts the tanning in the basket, or it gets the mallet again!

remembers me on the silence of the lambs film :D

**Comment by Rushor** [comment]
DB/Quest: Seeking Spiritual Aid - Update quest-end-event

* Islen Waterseernow uses the correct texts and playertargets
* Manifestation of Water now uses the correct text
* update orientation of Islen Waterseer at the end of the event to the spawnorientation again
* closes: https://github.com/dalaranwow/dalaran-wow/issues/4722
* commit: d4c2973480521e343f69af0ed841a1e45ff52127

http://i.imgur.com/eAkzxYp.jpg + http://i.imgur.com/t6746UA.jpg

The issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by TeridanDW** [comment]
Works as intended on PTR.

Note: If noone is around while the event is happening, the creatures just delays their action and once someone comes in the area event continues.
